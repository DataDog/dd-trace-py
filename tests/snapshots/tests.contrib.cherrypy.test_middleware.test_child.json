[[
  {
    "name": "cherrypy.request",
    "service": "test.cherrypy.service",
    "resource": "GET /child",
    "trace_id": 0,
    "span_id": 1,
    "parent_id": 0,
    "type": "web",
    "meta": {
      "_dd.p.dm": "-0",
      "http.method": "GET",
      "http.request.headers.host": "127.0.0.1:54583",
      "http.status_code": "200",
      "http.url": "http://127.0.0.1:54583/child",
      "runtime-id": "8cbf3566774347d39dcfc241879cbf84"
    },
    "metrics": {
      "_dd.agent_psr": 1.0,
      "_dd.top_level": 1,
      "_dd.tracer_kr": 1.0,
      "_sampling_priority_v1": 1,
      "system.pid": 54385
    },
    "duration": 2601000,
    "start": 1631828602147829000
  },
     {
       "name": "child",
       "service": "test.cherrypy.service",
       "resource": "child",
       "trace_id": 0,
       "span_id": 2,
       "parent_id": 1,
       "meta": {
         "a": "b"
       },
       "duration": 47000,
       "start": 1631828602147987000
     }]]
