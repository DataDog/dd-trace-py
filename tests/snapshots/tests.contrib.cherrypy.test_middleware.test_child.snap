[[{"name" "cherrypy.request"
   "service" "test.cherrypy.service"
   "resource" "GET /child"
   "type" "web"
   "error" 0
   "span_id" 0
   "trace_id" 0
   "parent_id" nil
   "start" 1612565466747391000
   "duration" 3020000
   "meta" {"runtime-id" "1e4bde87e00244cabc8e80e84a42ecb7"
           "http.method" "GET"
           "http.url" "http://127.0.0.1:54583/child"
           "http.status_code" "200"
           "http.request.headers.host" "127.0.0.1:54583"}
   "metrics" {"_dd.agent_psr" 1.0
              "_sampling_priority_v1" 1
              "_dd.tracer_kr" 1.0
              "system.pid" 13276}}
      {"name" "child"
       "service" "test.cherrypy.service"
       "resource" "child"
       "error" 0
       "span_id" 1
       "trace_id" 0
       "parent_id" 0
       "start" 1612565466747971000
       "duration" 98000
       "meta" {"a" "b"}}]]
